app/models/review.rb

```diff
@@ -0,0 +1,34 @@
+class Review < ApplicationRecord
+  include AASM
+
+  belongs_to :book
+  belongs_to :user
+
+  validates :title, :text,
+            presence: true,
+            format: { with: /\A[a-zA-Z0-9 !#\$%&'\*\+-=\/\?\^\_`\{\}\|~\.]+\z/,
+                      message: I18n.t('models.review.title_symbols') }
+  validates :title, length: { maximum: 79 }
+  validates :text, length: { maximum: 499 }
+
+  aasm column: 'status' do
+    state :unprocessed, initial: true
+    state :approved, :rejected
+
+    event :approve do
+      transitions from: :unprocessed, to: :approved
+    end
+
+    event :reject do
+      transitions from: :unprocessed, to: :rejected
+    end
+  end
+
+  scope :new_, -> do
```

Comment:
@b-artem first of all, really weird name for scope. Why do you need it? aasm provides helper methods, so you can use `Review.unprocessed`

UPDATED: I can't find usage of this scope, is it true?

Reply:
This scope is used in rails_admin.rb:
```ruby
  config.model 'Review' do
    list do
      scopes [:new_, :processed]
      ...
```
The idea behind that name was to meet the following requirement:
* Admin sees the list of new reviews(unprocessed) in **New** tab.

It's easy to change model or field label in Rails Admin, but I still can't figure out how to change scope label from 'Unpocessed' to 'New'